datacats/scripts/compile_less.sh

```diff
@@ -1,2 +1,9 @@
 #!/bin/bash
-lessc $1 > $2
+
+
+lessc /project/target/ckan/ckan/public/base/less/main.less \
+> /project/target/ckan/ckan/public/base/css/main.debug.css
```

Comment:
should we be generating the main.css too? I think when debug is off that's the css file that's actually used